charts/voltdb/values.yaml

```diff
@@ -0,0 +1,405 @@
+  ## @param voltdb.importers Importer configuration for voltdb
+  ##
+  importers: []
+  # - type: kafka
+  #   enabled: true
+  #   format: csv
+  #   version: 10
+  #   properties:
+  #     - name: topics
+  #       value: employees
+  #     - name: procedure
+  #       value: EMPLOYEE.insert
+  #     - name: brokers
+  #       value: kafkasvr:9092
+
+  ## @param voltdb.exporters Exporter configuration for voltdb
+  ##
+  exporters: []
+  # - type: kafka
+  #   enabled: true
+  #   target: target
+  #   properties:
+  #     - name: bootstrap.servers
+  #       value: kafkasvr:9092
+  #     - name: topic.prefix
+  #       value: employees
+  #     - name: skipinternals
+  #       value: true
+  #     - name: acks
+  #       value: 1
+
+  ## @param voltdb.sitePerHost Number of unique VoltDB sites that are created on each physical database server
+  ##
+  sitePerHost: 1
+  ## @param voltdb.kfactor Number of times partitions will be replicated
+  ##
+  kfactor: 0
+  ## @param voltdb.snapshots.enabled Enable automated snapshots?
+  ## @param voltdb.snapshots.frequency Frequency for automated snapshots
+  ## @param voltdb.snapshots.prefix Prefix for snapshots
+  ## @param voltdb.snapshots.retentionPeriod Retention period for snapshots in days
+  ##
+  snapshots:
+    enabled: false
+    frequency: 24h
+    prefix: AUTO
+    retentionPeriod: 2
+  ## @param voltdb.commandLog.enabled Enable command logging?
+  ## @param voltdb.commandLog.synchronous Should the logging be synchronous?
+  ## @param voltdb.commandLog.logSize Disk space pre allocated for storing the logs on disk in MBs
+  ## @param voltdb.commandLog.frequency.time Write logs after every <time> milliseconds
+  ## @param voltdb.commandLog.frequency.transactions Write logs after every <transactions> transactions
+  ##
+  commandLog:
+    enabled: false
+    synchronous: false
+    logSize: 1024
+    frequency:
+      time: 200
+      transactions: 1000
+  ## @param voltdb.heartbeat.timeout Heartbeat timeout between nodes
+  ##
+  heartbeat:
+    timeout: 90
+  ## @param voltdb.systemSettings.tempTables.maxSize Max size of temporary tables in MBs
+  ## @param voltdb.systemSettings.snapshot.priority Snapshot priority
+  ## @param voltdb.systemSettings.elastic.duration Time each rebalance transaction will take, in milliseconds
+  ## @param voltdb.systemSettings.elastic.throughput Number of MBs per second that will be processed by the rebalance transactions
+  ## @param voltdb.systemSettings.query.timeout Query timeout in milliseconds
+  ## @param voltdb.systemSettings.procedure.logInfo Log info whenever a task runs for more than specified time in milliseconds
+  ## @param voltdb.systemSettings.resourceMonitor.frequency Frequency at which resource limits are checked
+  ## @param voltdb.systemSettings.resourceMonitor.memoryLimit.size Maximum allowable resident set size
+  ## @param voltdb.systemSettings.resourceMonitor.memoryLimit.alert Alert thresold
+  ##
+  systemSettings:
```

Comment:
Are there other settings that are not exposed by chart but are available? if yes, we need to have a provision to pass them as is like we are doing in other charts

Reply:
There are other available [configurations](https://docs.voltdb.com/UsingVoltDB/ConfigStructure.php). 
`deployment.xml` is not generated using `voltdb.deployment` in `values`. Some default configurations are set in `values.yaml` and additional configurations can be provided.